app/templates/index.hbs

```diff
@@ -33,6 +33,15 @@
           English Service
         </div>
       </div>
+      <div class="col-sm-4 group" {{action "viewEnglishService"}}>
+        <img src={{cloudinary-fetch url="https://mcac.s3.amazonaws.com/groups/5c7d76a8-d195-4907-951d-5b3fb00c438d.jpg"
+                                    width=200
+                                    radius="max"
+                                    height=200 crop="fill"}} />
+        <div class="description">
+          More Coming Soon... <i class="icon-cma logo"></i>
```

Comment:
I don't think we need the logo here. I meant that maybe the circle could be the logo... but even that probably doesn't look good. For now, just remove this logo.